### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by weirdan**
## Enable PHP 7.3 builds
Now that PHP 7.3 is released `nightly` points to `7.4-dev`, so Psalm was lacking proper 7.3 builds

## Speed up code coverage job runs
Using technique from [article by Sebastian Bergmann and Sebastian Heuer](https://thephp.cc/news/2019/01/faster-code-coverage) code coverage job runs are now ~30% faster (compare https://travis-ci.com/weirdan/psalm/builds/97308745 vs https://travis-ci.org/vimeo/psalm/builds/479228801)

### CODE REVIEW

**Comment by muglug** [comment]
Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -50,24 +51,30 @@ matrix:
...
       env: XDEBUG="true"
       install:
         - composer $DEFAULT_COMPOSER_FLAGS update
         - composer require --dev php-coveralls/php-coveralls
+        - mkdir -p build
+      before_script:
+        - vendor/bin/phpunit --dump-xdebug-filter build/xdebug-filter.php
+      script:
+        - vendor/bin/phpunit --prepend build/xdebug-filter.php
+        - ./psalm --find-dead-code
```
Comment (muglug) [inline .travis.yml]: Is this necessary?

Comment (weirdan) [reply .travis.yml]: What specifically? Psalm run? Or are you referring to xdebug-filter linex?

**Comment by muglug** [comment]
Yeah, did Psalm run in the code coverage step before?

**Comment by weirdan** [comment]
> Yeah, did Psalm run in the code coverage step before?

Yes, code coverage run was also used as PHP7.2/high-deps runs (omitted from the regular test run)

**Comment by weirdan** [comment]
See https://travis-ci.org/vimeo/psalm/jobs/479228810#L994

**Pull request merged**